addusers.py

```diff
@@ -43,6 +43,7 @@
 #local
 import message
 import spreadsheet
+from moc_utils import get_absolute_path
```

Comment:
Not yet, but we should.  I opened a feature request for it, we can give this task to an intern maybe: https://github.com/CCI-MOC/moc-openstack-tools/issues/34